tests/env/components/mock_service_control.go

```diff
@@ -259,6 +259,35 @@ func (m *MockServiceCtrl) GetRequests(n int) ([]*utils.ServiceRequest, error) {
 	return r, nil
 }
 
+func isCheckOnlyQuota(in *utils.ServiceRequest) bool {
+	if in.ReqType != utils.QuotaRequest {
+		return false
+	}
+	got, err := utils.UnmarshalQuotaRequest(in.ReqBody)
+	// If not QuotaRequest, not to ignore
+	if err != nil {
+		return false
+	}
+	return got.AllocateOperation.QuotaMode == scpb.QuotaOperation_CHECK_ONLY
+}
+
+// GetRequestsWithoutCheckOnlyQuota returns a slice of requests received.
+func (m *MockServiceCtrl) GetRequestsWithoutCheckOnlyQuota(n int) ([]*utils.ServiceRequest, error) {
```

Comment:
Ok, so there is no easy way to verify check only quota requests? Because it happens every second, it is time dependent

Reply:
Yeap,  number of calls are time dependent, it is kind of tricky to verify it.

Reply:
Could you make `GetRequestsWithoutCheckOnlyQuota`  return if the `CheckOnlyQuota` exist?